services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/FlowRulesSpec.groovy

```diff
@@ -619,19 +618,24 @@ class FlowRulesSpec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
-    @Ignore("sync is not working yet, fix in pr2591")
     @Tags(HARDWARE)
     def "Able to synchronize rules for a flow with VXLAN encapsulation"() {
-        given: "Two active Noviflow switches"
-        def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find { it.src.noviflow && it.dst.noviflow }
+        given: "Two active not neighboring Noviflow switches"
+        def switchPair = topologyHelper.getAllNotNeighboringSwitchPairs().find { swP ->
+            swP.src.noviflow && swP.dst.noviflow && swP.paths.find { path ->
+                pathHelper.getInvolvedSwitches(path).find { it.noviflow }
```

Comment:
should be `pathHelper.getInvolvedSwitches(path).every { it.noviflow }` if you want a path with all-novi switches

Reply:
fix